specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -1626,6 +1626,11 @@
           },
           "default": "Microsoft.Storage"
         },
+        "requireInfrastructureEncryption": {
+          "type": "boolean",
+          "x-ms-client-name": "RequireInfrastructureEncryption",
+          "description": "A boolean indicating whether or not the service applies a secondary layer of encryption with platform managed keys for data at rest."
```

Comment:
Is there a feature spec this it? 
What's the difference of the storage account when set the value to true or false? And if user don't set it, what's the default value?
Does the property only support on create, but not support on update account?